build.gradle

```diff
@@ -47,8 +47,12 @@ repositories {
 bootJar {
     baseName = 'edm-zuul-proxy'
     mainClassName= 'edu.uw.edm.edmzuulproxy.EdmZuulProxyApplication'
+}
 
+springBoot {
+    mainClassName = 'edu.uw.edm.edmzuulproxy.EdmZuulProxyApplication'
```

Comment:
Without this change, I receive this error when running locally:
`
Execution failed for task ':bootRun'.
> Unable to find a single main class from the following candidates [edu.uw.edm.CertificateManagementApplication, edu.uw.edm.edmzuulproxy.EdmZuulProxyApplication]
`